test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -68,6 +69,7 @@ public void CanChangeEngineFromScriptEditorDropDown()
             var engineAfterChange = engineSelectorComboBox.SelectedItem = engineChange;
 
             // Assert
+            Assert.AreEqual(engineSelectorComboBox.Visibility, Visibility.Visible);
```

Comment:
Previous this test pass no matter debug mode is on/off which is worrisome. Adding visibility check